__init__.py

```diff
@@ -310,12 +303,29 @@ def execute(self, _context):
         print("Starting Import")
         import webbrowser
         print("Starting Login")
-        if thangs_api.bearer == "":
+        
+        bearer_DIR = os.path.join(os.path.dirname(__file__), 'bearer.json')

```

Comment:
Let's have this be something like `bearer_location` instead.  It's a file, not a directory, so the name could be confusing.